### PR TITLE
Blogpost about UFRN Nextflow graduate course

### PR DESCRIPTION
**Pull request by mribeirodantas**

### CODE REVIEW

**Review by ewels — APPROVED** [review]

**Review by ewels — CHANGES_REQUESTED** [review]
Needs a social share image before publishing

File: content/blog/2023/nextflow-goes-to-university.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+title=Nextflow goes to university!
+date=2023-07-30
+type=post
+description=Helpful tips for troubleshooting out-of-memory errors in containerized execution environments.
+image=img/to-be-decided.jpg
```
Comment (ewels) [inline content/blog/2023/nextflow-goes-to-university.md:5]: Needs this 😊

File: content/blog/2023/nextflow-goes-to-university.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+title=Nextflow goes to university!
+date=2023-07-30
+type=post
+description=Helpful tips for troubleshooting out-of-memory errors in containerized execution environments.
```
Comment (ewels) [inline content/blog/2023/nextflow-goes-to-university.md:4]: Needs updating

File: content/blog/2023/nextflow-goes-to-university.md (hunk 1 above)
Comment (mribeirodantas) [reply content/blog/2023/nextflow-goes-to-university.md:5]: Yup, I left this for last haha

File: content/blog/2023/nextflow-goes-to-university.md (hunk 2 above)
Comment (mribeirodantas) [reply content/blog/2023/nextflow-goes-to-university.md:4]: This I missed indeed 🤣

File: content/blog/2023/nextflow-goes-to-university.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+title=Nextflow goes to university!
+date=2023-07-34
+type=post
+description=Seqera partnership with universities allows students to learn Nextflow and get credits for that!
```
Comment (ewels) [inline content/blog/2023/nextflow-goes-to-university.md:4]:
```suggestion
description=Seqera partnership with universities to help students to learn Nextflow
```

**Review by ewels — APPROVED** [review]

File: content/blog/2023/nextflow-goes-to-university.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+title=Nextflow goes to university!
+date=2023-07-34
```
Comment (ewels) [inline content/blog/2023/nextflow-goes-to-university.md:2]:
```suggestion
date=2023-07-24
```

Comment (mribeirodantas) [reply content/blog/2023/nextflow-goes-to-university.md]: Oooops thanks 😅

**Pull request merged**